scripts/remove-emojis.lua

```diff
@@ -0,0 +1,24 @@
+local demojify = require("Demojify")
```

Comment:
Did you try this? 

```suggestion
local demojify = require("lib.Demojify")
```

It would be nice ideally to have this work in a way that even if people git clone'd the repo, they'd be able to use with minimal setup?

Reply:
Yes, this works 🎉
I had to adjust the directory the file is copied to in the Dockerfile, now it works both locally, and in the GH Action:
```
pandoc --metadata=project:"panvimdoc" --lua-filter scripts/skip-blocks.lua --lua-filter scripts/include-files.lua --lua-filter scripts/remove-emojis.lua -t scripts/panvimdoc.lua ../nvim/README.md -o "doc/panvimdoc.txt"
```
and here's the [Action](https://github.com/nekowinston/doctest/runs/7489739256?check_suite_focus=true) and the [result](https://github.com/nekowinston/doctest/blob/main/doc/catppuccin.txt).